apps/nextra/pages/en/build/sdks/ts-sdk/simulating-transactions.mdx

```diff
@@ -0,0 +1,157 @@
+# Simulating transactions
+
+Simulating transactions allows you to preview the cost and effect of submitting a transaction without paying fees. 
+You can use this to estimate fees, test a transaction, or to check what the output might be. 
+
+To simulate a transaction, you must pass in a transaction and which account would be the signer:
+
+```ts filename="simulate-a-transaction.ts"
+import {
+    Account,
+    Aptos,
+    AptosConfig,
+    Network,
+} from "@aptos-labs/ts-sdk";
```

Comment:
Should we remote insert these examples @hariria?

Reply:
I'll note that this example is not quite the same as the one in the `building-transactions.mdx`. It borrows most of it, but needs to be a bit different so that it can be copy + pastable. (Ex. The call to `example(...)` happens way earlier, and I removed some of the print statements :) )

Reply:
Yeah, basically what I mean is, if we have simple examples, we might want to put them in the code as... well simple examples.

Either way I'm good with this for now!